### ISSUE TITLE
feature request: I would like for Spoon to be able to merge the output of several reports into one output

### ISSUE THREAD

**Issue by todorus** [square/spoon#402]
**feature request**
I would like for Spoon to be able to merge the output of several reports into one output

**background**
Spoon is normally used in a situation where the app under test is the whole context. I am in the situation of setting up test tooling for an embedded Android device, where the device is the context and the app is just a component. 

The solution that I'm looking into, is to use a script to run some instrumentation tests and measure the outputs from UI and the other (hardware) components. So the instrumentation tests will not always be used to test something, but to simulate user interaction via the app.

We would love to have the excellent reporting features that Spoon offers, but Spoon naturally assumes that it is **the** testrunner and I end up with multiple folders of Spoon reports. Any such report only holds an overview of a single or few tests. This negates the feature we really love about Spoon: to provide a single overview of the result of the test.

**Comment by todorus** [comment]
I've added this story to our backlog as well and I expect to be able to pick it up next sprint.

It seems to me that my lowest hanging fruit is the result.json file:
1. merge the result.json files
2. solve the conflicts 
3. feed it to the SpoonSummary class

**Comment by edenman** [comment]
I don't think this should be part of the main Spoon workflow.  One way you could go:
- new command-line tool to merge result.json files
- expose `HtmlRenderer` as a new maven target (or separate executable?) that only takes a `result.json` path?

@JakeWharton thoughts?

**Comment by JakeWharton** [comment]
I don't have strong opinions but this has come up before. I like the idea
of a second main method that only does HTML rendering from the JSON files.
JSON merging can be done with existing CLI tools like `jq`.
